server/app/services/applications/ProgramAdminApplicationService.java

```diff
@@ -2,6 +2,7 @@
 
 import static com.google.common.base.Preconditions.checkNotNull;
 
+import autovalue.shaded.com.google.common.annotations.VisibleForTesting;
```

Comment:
```suggestion
import com.google.common.annotations.VisibleForTesting;
```
(blocking) Oh looks like the wrong package name was used.